diff --git a/node_modules/node-pty/binding.gyp b/node_modules/node-pty/binding.gyp
index f21c7df..9c77df9 100644
--- a/node_modules/node-pty/binding.gyp
+++ b/node_modules/node-pty/binding.gyp
@@ -5,8 +5,11 @@
     ],
     'conditions': [
       ['OS=="win"', {
+        'defines': [
+          'NDEBUG'
+        ],
         'msvs_configuration_attributes': {
-          'SpectreMitigation': 'Spectre'
+          'SpectreMitigation': 'false'
         },
         'msvs_settings': {
             'VCCLCompilerTool': {
diff --git a/node_modules/node-pty/deps/winpty/src/winpty.gyp b/node_modules/node-pty/deps/winpty/src/winpty.gyp
index 1ac5758..9a3553e 100644
--- a/node_modules/node-pty/deps/winpty/src/winpty.gyp
+++ b/node_modules/node-pty/deps/winpty/src/winpty.gyp
@@ -41,7 +41,7 @@
                 '-luser32',
             ],
             'msvs_configuration_attributes': {
-                'SpectreMitigation': 'Spectre'
+                'SpectreMitigation': 'false'
             },
             'msvs_settings': {
                 # Specify this setting here to override a setting from somewhere
@@ -143,7 +143,7 @@
                 '-luser32',
             ],
             'msvs_configuration_attributes': {
-                'SpectreMitigation': 'Spectre'
+                'SpectreMitigation': 'false'
             },
             'msvs_settings': {
                 # Specify this setting here to override a setting from somewhere
